src/api/app/assets/stylesheets/webui/responsive_ux.scss

```diff
@@ -1,3 +1,9 @@
+// Variables
+// TODO: move to app/assets/stylesheets/webui/bootstrap_variables/sizes.scss
```

Comment:
@saraycp is there are reason to leave it here with the TODO comment, or is it a left over?

Reply:
The idea is to move this part to the correct file as soon as we move it out of the beta program.